password_security/controllers/main.py

```diff
@@ -44,6 +45,7 @@ def web_login(self, *args, **kw):
             request.params['password']
         )
         if not uid:
+            request.uid = old_uid
```

Comment:
Hello @lasley , can you explain more?
When website is loaded, current user will be public user, so `old_uid` will be `public_user`.
In the other hand, if last user  logged out, current user will be back to `public_user` again`.
Thank you!